app/follow-list/page.tsx

```diff
@@ -0,0 +1,47 @@
+"use client";
+
+import { setInstanceToken } from "@/apis";
+import Error from "@/components/common/Error";
+import FollowList from "@/components/follow-list/FollowList";
+import UserProfile from "@/components/follow-list/UserProfile";
+import FollowListSkeleton from "@/components/follow-list/skeletonUi/FollowListSkeleton";
+import { useGetCombinedUserInfo } from "@/hooks/user/useGetCombinedUserInfo";
+import { getSessionStorageHandler } from "@/utils/getSessionStorageHandler";
+import * as styles from "../../styles/follow-list/FollowListPageContainerStyle.css";
+
+const FollowListPage = () => {
+  const token: string | null =
+    typeof window !== "undefined"
+      ? getSessionStorageHandler().getItem("token")
+      : null;
+  const isHasToken =
+    typeof window !== "undefined"
+      ? getSessionStorageHandler().hasItem("token")
+      : false;
```

Comment:
ASK: 혹시 null과 false를 반환하는 형태로 구현하신 이유가 따로 있을까요!?
조건부 처리를 할때 빈 값을 반환하는게 좋은 패턴은 아니라고 들어서요..!
특별한 이유가 없다면 아래처럼 수정해봐도 좋을 것 같습니당 😺😺

```suggestion
  const token: string = typeof window !== "undefined"
      && getSessionStorageHandler().getItem("token");
  const isHasToken = typeof window !== "undefined"
      && getSessionStorageHandler().hasItem("token");
```

Reply:
좋은 패턴 제안 감사합니다!

token 값은 값이 있거나(string) 없을 때는 명시적으로 null 값이 되도록 타입을 지정해주었는데요, 삼항연산자로 window가 undefined 일 때 null 임을 정의해주지 않으면 (`&&`연산자 사용) 값이 false(boolean type)으로 인식되어 타입 에러가 나더라구요!

반면 isHasToken은 boolean 값의 type을 갖는 값이기 때문에 제안 주신것처럼 삼항 연산자가 아닌 && 연산자로 구현해도 문제가 없었습니다!

따라서 token은 원래 로직대로 삼항 연산자로 winodw값이 잡히지 않을시 null 값을 명시해주고,
isHasToken은 `&&` 연산자로 window 값이 잡히지 않을 시 false의 boolean 타입이 추론되도록 로직 수정했습니다 ~